pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/SimpleFilter.java

```diff
@@ -49,22 +54,30 @@
  *     @Property(name="service.ranking", intValue=1)
  * })
  */
-//@SlingFilter(order=1, description="A Simple Filter")
-//@Property(name="service.vendor", value="The Apache Software Foundation")
+@Component
+@SlingServletFilter(scope = {SlingServletFilterScope.REQUEST},
+                    methods = {"GET","HEAD"})
```

Comment:
GET  and HEAD will do for now, but we **might** need to extend this to other methods.

Reply:
i added pattern value to limit when this filter will run:
@SlingServletFilter(scope = {SlingServletFilterScope.REQUEST},
                    pattern = "/api/.*",
                    methods = {"GET","HEAD", "OPTIONS"})